app/presenters/cost_of_living_landing_page_presenter.rb

```diff
@@ -28,10 +28,10 @@ def link_clicked_track_data(track_action:, href:)
 private
 
   def internal_link?(link)
-    link.starts_with?("https://www.gov.uk/")
+    link.starts_with?("/")
   end
 
   def slug_for_href(href)
```

Comment:
I think we can now delete this method entirely as it's not making any changes and on line 23 have simply `track_label: href,`